Source/websocket/WebSerializer.cpp

```diff
@@ -359,8 +359,17 @@ namespace Web
         return (Core::EnumerateType<type>(value).Data());
     }
 
+    bool endsWithCaseInsensitive(const string& mainStr, const string& toMatch)
```

Comment:
Minor, but to keep code consistent, Methods are always PascalCase, so could you please name it:
EndsWithCaseInsensitive

Reply:
Done